src/Klarna.OrderManagement/Klarna.OrderManagement.v3.csproj

```diff
@@ -258,6 +257,9 @@
       <Name>Klarna.Common.v3</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Refunds\" />
```

Comment:
What is this folder for?

Reply:
I've removed interface IRefundBuilder - this is a leftover folder. 
I can remove it.

Reply:
This still needs to be removed